force-app/main/default/lwc/updatePaymentMethodModal/updatePaymentMethodModal.html

```diff
@@ -0,0 +1,27 @@
+
+<template>
+
+  <template if:true={open}>
+      <section role="dialog" tabindex="-1" aria-modal="true" aria-labelledby="modal-heading-01" class="slds-modal slds-fade-in-open">
```

Comment:
Same for this custom modal. Javascript focus trapping should be implemented.

Reply:
Added!